velox/docs/develop/memory.rst

```diff
@@ -539,7 +539,7 @@ between queries by adjusting their memory pool’s capacities accordingly (see
 
 The *MemoryArbitrator* is defined to support different implementations for
 different query systems. As for now, we implement *SharedArbitrator* for both
-Prestissimo and Prestissimo-on-Spark. `Gulten <https://github.com/oap-project/gluten>`_ implements its own memory
+Prestissimo and Prestissimo-on-Spark. `Gluten <https://github.com/oap-project/gluten>`_ implements its own memory
```

Comment:
How about changing the gluten link to `https://github.com/apache/incubator-gluten` ?